webpack.config.js

```diff
@@ -81,18 +80,18 @@ var config = {
       {
```

Comment:
You should use `module.rules` instead of old `module.loaders` syntax.
https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules